Formula/crystal-lang.rb

```diff
@@ -55,6 +70,8 @@ def install
     macho = MachO.open("#{buildpath}/boot/embedded/bin/crystal")
     macho.change_dylib("/usr/local/opt/libevent/lib/libevent-2.0.5.dylib",
                        "#{buildpath}/vendor/libevent/lib/libevent-2.0.5.dylib")
+    macho.change_dylib("/usr/local/opt/bdw-gc/lib/libgc.1.dylib",
```

Comment:
Could this not be changed to do the equivalent of

```sh
libevent_lib=$(brew --prefix libevent)/lib
libevent=${libevent_lib}/$(readlink ${libevent_lib}/libevent.dylib)
libgc_lib=$(brew --prefix bdwgc)/lib
libgc=${libgc_lib}/$(readlink ${libgc_lib}/libgc.dylib)
```

And then doing the `macho.change_dylib…` with the resulting values, since we currently depend on those libraries, anyway?

(But thank you for this fix/workaround, regardless.)